### DISCUSSION TITLE
Another project "ivoyager"

### DISCUSSION THREAD

**Discussion by Valinakova** [Hoimar/Planet-Generator#12] [Ideas]
Hey, I found another project that might have useful code to add to this addon.

It's ivoyager, have you heard of it? Maybe there is code in there that might be useful. It is under another type of permissive license, Apache 2.0. A little different from MIT but shouldn't be too different. Same attribution requirements, but if you modify I believe they just require you to state that you modified the code base.  https://github.com/ivoyager/planetarium

**Comment by Hoimar** [comment]
Thanks, I appreciate it.
Yes, I know ivoyager and already had a quick look at the code recently. It's a really nice project, yeah. Not quite sure which parts might be useful for this project, perhaps the origin shifting, dynamic camera adjusting etc. I'll look into it.

**Reply by Valinakova** [reply]
Yeah that was what I was thinking. Origins shifting would be useful even for double float precision in 4.0.
